### PR TITLE
vscode: update to 1.49.1 + switch to electron9. [ci skip]

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/vscode/template [hunk 1]
```diff
@@ -46,7 +45,7 @@ do_build() {
 	export NODE_OPTIONS="${_mem_limit}"
 	/usr/lib/node_modules/npm/bin/node-gyp-bin/node-gyp install \
 		--target=$_electronver \
-		--tarball=/usr/include/electron7/node_headers.tar.gz
+		--tarball=/usr/include/electron9/node_headers.tar.gz
```
Comment (fosslinux) [inline srcpkgs/vscode/template:48]: I think this could be programatically set, `"${_electronver%%.*}` i think

**Review by fosslinux — CHANGES_REQUESTED** [review]

File: srcpkgs/vscode/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/vscode/template:48]: Absolutely. Thx. Gonna update this ASAP.

File: srcpkgs/vscode/template [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-version=1.49.0
+version=1.49.1
 revision=1
-# XXX: please update to electron9 next time you rebuild this package
-_electronver=7.3.3
+_electronver=9.2.1
 hostmakedepends="pkg-config python nodejs-lts-10 yarn tar git"
-makedepends="libxkbfile-devel libsecret-devel electron7"
+makedepends="libxkbfile-devel libsecret-devel electron9"
 depends="libXtst libxkbfile nss dejavu-fonts-ttf xdg-utils ripgrep electron7"
```
Comment (Johnnynator) [inline srcpkgs/vscode/template:8]:
```suggestion
depends="libXtst libxkbfile nss dejavu-fonts-ttf xdg-utils ripgrep electron9"
```
You forgot to update the runtime dep.

**Review by Johnnynator — CHANGES_REQUESTED** [review]

File: srcpkgs/vscode/template (hunk 1 above)
Comment (Johnnynator) [reply srcpkgs/vscode/template:48]: You could also only specify the major version as variable and read the full version from `/usr/lib/electron${electon_major}/version`

**Pull request merged**